lambdas/functions/control-plane/src/lambda.test.ts

```diff
@@ -216,7 +216,7 @@ describe('Test job retry check wrapper', () => {
     const error = new Error('Error handling retry check.');
     mock.mockRejectedValue(error);
 
-    expect(await jobRetryCheck(sqsEvent, context)).resolves;
+    await expect(jobRetryCheck(sqsEvent, context)).resolves.not.toThrow('Error handling retry check.');
```

Comment:
We checking the promise resolves without an error.
```suggestion
    await expect(jobRetryCheck(sqsEvent, context)).resolves.not.toThrow();
```